app/models/article.rb

```diff
@@ -1,5 +1,15 @@
 class Article < ApplicationRecord
-    validates :title, presence: true, length: { minimum: 6, maximum: 100 }
-    validates :description, presence: true, length: { minimum: 10, maximum: 300 }  
-    #has_many :comments
-  end
\ No newline at end of file
+  has_many :comments
+  validates :title, presence: true, length: { minimum: 6, maximum: 100 }
+  validates :description, presence: true, length: { minimum: 10, maximum: 300 }    
+
+
+  scope :relevant, -> { where("expiration_date > ? ", DateTime.now) }
+  scope :expired, -> { where("expiration_date < ? ", DateTime.now) }
+
+  STATUS = {
```

Comment:
Remove it. To implement statuses use enum. 
E.g
`enum status: { opened: 0, expired: 1, closed: 2 }`